public/index.html

```diff
@@ -20,6 +20,8 @@
     <link rel="icon" href="%PUBLIC_URL%/images/bcid-favicon-16x16.png" sizes="16x16" type="image/png" />    
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
 
+    <script src="%REACT_APP_KEYCLOAK_AUTH_URL%/js/keycloak.js"></script>
```

Comment:
Fine for now, but we're going to have to solve async loading at some point - I'll write a ticket for it.

Reply:
By putting this script tag in `<head>`tag we don't have to worry about loading issue because this will be loaded first before React related scripts

Reply:
It is going to slow down the overall loading of the page though. On second thought, it's not even worth a ticket - but we can get faster loading overall by loading it asynchronously and putting the wait in where we're actually needing to auth. BUT - REALLY - the file is going to be cached by the browser, so never mind - too much effort for speeding up the initial load.